backend/test_observer/controllers/issues/models.py

```diff
@@ -72,36 +70,6 @@ class IssueTestResultAttachmentRulePatchRequest(BaseModel):
     enabled: bool | None = None
 
 
-class IssueTestResultAttachmentRuleResponse(BaseModel):
```

Comment:
Moved to `shared_models.py` so it can be included in `MinimalIssueTestResultAttachmentResponse`